lib/Main.js

```diff
@@ -568,7 +569,7 @@ Main.prototype.run = function(port) {
         })
     }).then((entries) => {
         entries.forEach((entry) => {
-            console.log("Ignoring LEGACY room entry in room-store.db", entry);
+            log.error("Ignoring LEGACY room entry in room-store.db", entry);
```

Comment:
warn?

Reply:
I'd call it an error since it shouldn't really ever happen. And it's only going to be emitted on startup.